.github/workflows/ci-cd.yml

```diff
@@ -180,17 +180,18 @@ jobs:
 
     steps:
     - name: Retrieve the project source from an sdist inside the GHA artifact
-      uses: re-actors/checkout-python-sdist@release/v1
+      uses: re-actors/checkout-python-sdist@release/v2
       with:
         source-tarball-name: >-
           ${{ needs.build-pure-python-dists.outputs.sdist-filename }}
         workflow-artifact-name: >-
           ${{ needs.pre-setup.outputs.dists-artifact-name }}
     - name: Download distributions
-      if: ${{ !endsWith(matrix.pyver, '-dev') && true || false }}
-      uses: actions/download-artifact@v3
+      if: ${{ !endsWith(matrix.pyver, '-dev') }}
```

Comment:
This change doesn't seem necessary. Best submit it separately.

Reply:
Yeah, I though that it could go in since it's a simple one-line improvement. Removed.